Trimble.Modus.Components/Controls/TMInput/CustomInput.xaml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"

```

Comment:
Custom validation works in reverse. It shows `Invalid Text` for valid text and vice versa.
<img width="529" alt="Screen Shot 2023-06-13 at 10 37 54 AM" src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/2a13be8c-308f-4d8c-a573-ca273289423b">

Reply:
I had changed the validation as alphanumeric alone so its showing like that. Shall I change that validation regex?

Reply:
Oh, I didn't know that. Can you update the helper text with what is a valid text? Like 'Enter a valid email' or in your case 'Enter only alphanumeric character'?